python_scripts/parameter_tuning_grid_search.py

```diff
@@ -104,10 +105,10 @@
 # ## Tuning using a grid-search
 #
 # Instead of manually writing the two `for` loops, scikit-learn provides a
-# class called `GridSearchCV` which implement the exhaustive search implemented
+# class called `GridSearchCV` which implements the exhaustive search we did
 # during the exercise.
```

Comment:
I guess "previous exercise" would help here (this is a pity we don't have ways to create links that work for all our setups, i.e. FUN-MOOC, JupyterBook, Jupyter notebook ...)